libcloud/storage/drivers/s3.py

```diff
@@ -1004,3 +1013,21 @@ class S3SAEastStorageDriver(S3StorageDriver):
     name = 'Amazon S3 (sa-east-1)'
     connectionCls = S3SAEastConnection
     ex_location_name = 'sa-east-1'
+
+
+class S3RGWOutscaleConnection(S3Connection):
+    pass
+
+
+class S3RGWOutscaleStorageDriver(S3StorageDriver):
+
+    def __init__(self, *args, **kwargs):
```

Comment:
Please explicitly declare all the arguments where possible instead of using *args and **kwargs.